src/Microsoft.TemplateEngine.Cli/PostActionProcessors/ProcessStartPostActionProcessor.cs

```diff
@@ -55,5 +59,21 @@ public bool Process(IEngineEnvironmentSettings settings, IPostAction actionConfi
 
             return allSucceeded;
         }
+
+        private static string TryGetExecutableFileFullPath(string executableFileName, string outputBasePath)
```

Comment:
Nit: having `Try` in method name imposes to have the following signature bool TrySmth (out object value), where value is returned in value parameter. `GetExecutableFileFullPath` might be better name

Reply:
Right, thank you, changed name to "ResolveExecutableFilePath".